diff --git a/node_modules/react-native-ble-peripheral/android/build.gradle b/node_modules/react-native-ble-peripheral/android/build.gradle
index 9b5638c..40c8a7b 100644
--- a/node_modules/react-native-ble-peripheral/android/build.gradle
+++ b/node_modules/react-native-ble-peripheral/android/build.gradle
@@ -15,8 +15,8 @@ android {
     }
 }
 dependencies {
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.android.support:appcompat-v7:23.0.1"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
 }
 
